src/components/VolunteerApplicationsList/index.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Box, InfiniteScroll, Text, Button,
+} from 'grommet';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { Alert } from 'grommet-icons';
+import Spinner from 'Components/Utils/Spinner';
+import UserProfileInfo from 'Components/Utils/UserProfileInfo';
+import PropTypes from 'prop-types';
+import orderServices from 'Api/orders.service';
+import ErrorModal from 'Components/Modals/ErrorModal';
+import SuccessModal from 'Components/Modals/SuccessModal';
+
+const VolunteerApplicationsList = ({ orderId }) => {
+  const loggedIn = useSelector((state) => state.logUser.loggedIn);
+  const history = useHistory();
+  const [voluntierApplications, setVoluntierApplications] = useState([]);
```

Comment:
```suggestion
  const [volunteerApplications, setVolunteerApplications] = useState([]);
```
Y lo mismo para todos los voluntier por volunteer

Reply:
Ok